timesketch/lib/sigma_util.py

```diff
@@ -129,7 +133,13 @@ def get_sigma_rules(rule_folder, sigma_config=None):
 
     blocklist_path = None
     ignore = get_sigma_blocklist(blocklist_path)
-    ignore_list = list(ignore["path"].unique())
+    ignore_list = list(ignore.loc[ignore['bad'] == 'bad']["path"].unique())
+    experimental_list = list(
+        ignore.loc[ignore['bad'] == 'experimental']["path"].unique()
+    )
+    to_be_used_in_analyzer = list(
```

Comment:
Is the attribute `bad` part of the Sigma spec? It feels like we are overloading this term, and IMO it is getting a bit difficult to reason about what `bad` vs `good` vs `experimental` means.

Maybe change this from `bad` --> `status` instead? And document what the different statuses means.

bad = Not able to parse to TS supported query
good = TS supported query 
experimental = Enabled in TS but not run automatically

Reply:
yes that make sense, the only reason I would avoid the experimental word is that it is used as a status in the Sigma rules itself and could be confusing

Reply:
ok changed it to status and introduced the term "exploratory" which seems more appropriate